pkg/ksym/ksym_test.go

```diff
@@ -168,11 +168,6 @@ ffffffff8f6d15c4 a not_in_order
 		addr2 + 1: "xfrm_km_lock",
 	}, syms)
 
-	require.Equal(t, map[uint64]string{
```

Comment:
nit: Maybe it won't be exact but we can still have equivalent test cases for this case.

Reply:
I would rather not re-add these as they are implementation details that should be fully covered by this test

https://github.com/parca-dev/parca-agent/blob/2668fa07c12821f9aeed7e1ec54a01aad765369b/pkg/ksym/ksym_test.go#L232-L243

Happy to augment that test or add more to make sure that the functionality is correct despite of the cache layer, if you think that the current tests don't fully cover it at the moment

Reply:
True, your point is valid.